lib/Model/RecentPage.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace OCA\Collectives\Model;
+
+class RecentPage {
+	protected string $collectiveName = '';
+	protected string $pageUrl = '';
+	protected string $title = '';
+	protected string $emoji = '🗒';
```

Comment:
I don't think we should set a default for `$emoji` - in order to allow special-handling of empty emojis in the dashboard widget code.

Reply:
TODO: remove the default (logic from above is sufficient)